mobile/Screen/EventDetailScreen.js

```diff
@@ -99,32 +291,117 @@ function EventDetailScreen({
     }
   };
 
+  // TO DO: add 'first' tuesdays, etc.
+  const daysOfWeek = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+  const dayOfWeek = daysOfWeek[day];
+  const weeksOfMonth = ['First', 'Second', 'Third', 'Fourth'];
+  const weekOfMonth = weeksOfMonth[week - 1];
+
+  const saveIcon = isEventSaved ? require('../assets/detailScreen/saved.png') : require('../assets/detailScreen/unsaved.png');
+
   return (
-    <ScrollView style={styles.container}>
-      <View style={styles.heading}>
-        <Text style={styles.h1}>{title}</Text>
-        <Switch value={isEventSaved} onValueChange={onPressToggleSavedEvent} title="Event Save Button" />
-      </View>
-      <Text onPress={onPressEvent}>{organizations}</Text>
-      <View style={styles.border} />
-      <View style={styles.square} />
-      <Text>{day}</Text>
-      <View style={styles.square} />
-      <Text>{time}</Text>
-      <View style={styles.square} />
-      <Text>{location}</Text>
-      <View style={styles.border} />
-      <Text style={styles.h2}>Event Description</Text>
-      <Text>
-        {' '}
-        {summary}
-        {' '}
-      </Text>
-      <Text style={styles.h2}>Pictures</Text>
+    <ScrollView contentContainerStyle={styles.container}>
       <Image
-        style={styles.image}
-        source={{ uri: url }}
+        source={image}
+        style={styles.coverImage}
       />
+      <View style={styles.overlay} />
+      <ScrollView style={styles.overlayContent} showsVerticalScrollIndicator={false}>
+        <View style={styles.flexContainer}>
+          <Text style={styles.title}>{title}</Text>
+          <TouchableOpacity style={[{ paddingTop: 5 }, { marginLeft: 45} ]} onPress={onPressToggleSavedEvent}>
+            <Image style={styles.saveIcon} source={saveIcon} />
+          </TouchableOpacity>
+        </View>
+        <TouchableOpacity style={styles.organizationContainer} onPress={() => onPressOrganization()}>
+          <Text style={styles.organizationText}>{organization}</Text>
+          <Image source={require('../assets/detailScreen/info.png')} style={styles.infoImage} />
+        </TouchableOpacity>
+        <ScrollView style={styles.info}>
+          <View style={styles.infoRow}>
+            <Image source={require('../assets/detailScreen/calendarIcon.png')} style={styles.infoIcon} />
+            {recurringMonthly && <Text style={styles.infoText}>{weekOfMonth} {dayOfWeek}{'s'}</Text>}
+            {!recurringMonthly && <Text style={styles.infoText}>{dayOfWeek}</Text>}
```

Comment:
if these are weekly occurances, do we also want to put "Fridays" instead of "Friday"?